packages/suite-native-navigation/src/TabBar.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { TouchableOpacity, View } from 'react-native';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { BottomTabBarProps } from '@react-navigation/bottom-tabs';
+import { Icon, IconType } from '@trezor/atoms';
+import { RouteTabs } from './routes';
+
+const tabBarStyle = prepareNativeStyle(utils => ({
+    height: 86,
+    backgroundColor: utils.colors.gray100,
+    borderTopColor: utils.colors.gray300,
+    borderTopWidth: utils.borders.widths.sm,
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    paddingTop: 20.5,
+    paddingLeft: 51.5,
+    paddingRight: 51.5,
+}));
+
+export const bottomTabOptions: Record<string, { icon: IconType }> = {
+    [RouteTabs.Home]: { icon: 'home' },
+    [RouteTabs.Accounts]: { icon: 'accounts' },
+    [RouteTabs.Action]: { icon: 'action' },
+    [RouteTabs.Prices]: { icon: 'prices' },
+    [RouteTabs.SettingsStack]: { icon: 'settings' },
+};
+
+export const TabBar = ({ state, descriptors, navigation }: BottomTabBarProps) => {
```

Comment:
I had the same question at first, but according to documentation:

> Note that you cannot use the useNavigation hook inside the tabBar since useNavigation is only available inside screens. You get a navigation prop for your tabBar which you can use instead:

Reply:
Magic JS 😅

Thanks for explaining